Congress-Class Frigate (boattest13)

Ragnuv opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Adds the Congress-Class Frigate to the roster, a pirate vessel heavily inspired on the Fallout 4 USS Constitution.

![boatos](https://user-images.githubusercontent.com/99061354/156663268-34c927f0-ae46-4cd8-b03a-e47e2f73b92d.PNG)

## Why It's Good For The Game

Adds a boat to boattest13

## Changelog
:cl:
add: Congress-Class Pirate vessel
fix: floors, some stuff
fix: buttons, medkits, line endings
fix: movement, gygax access, step_y/x
fix: airlocks, balance
add: holopad
/:cl:

Ragnuv committed:
add congress class

add congress class

Ragnuv committed:
add congress class

add congress class json file

Ragnuv committed:
fixes floors, fixes some more stuff

fix

Ragnuv committed:
hopefully

fixes

rye-rice reviewed (changes requested):
step_x and step_y need to be changed to pixel_x and pixel_y respectively

UltraNumeron commented:
from testing this i will note that the mech is  a really weird model that needs adminspawn access so you cant  replace its battery or anything

Ragnuv committed:
More fixes

Fixes movement, adds a ghetto fix to gygax's access, nukes step_y/x from orbit

Ragnuv commented:
> from testing this i will note that the mech is a really weird model that needs adminspawn access so you cant replace its battery or anything

> step_x and step_y need to be changed to pixel_x and pixel_y respectively

Issues should be adressed now.

Ragnuv committed:
balance changes + map changes

Mostly balance changes, switches the Sec, mining and engineering hardsuits for syndicate EVA suits, and changes shotguns for laser guns, switches captain's gear, adds a small holding cell next to the armory, fixes the airlock, +other smaller changes.

Im satisfied with the ship by this point, this should be it's final rendition for a while.

Ragnuv committed:
add

adds a holopad
